src/main.c

```diff
@@ -1157,6 +1167,67 @@ static yoml_t *load_config(const char *fn)
     return yoml;
 }
 
+static yoml_t *resolve_tag(const char *tag, yoml_t *node, void *cb_arg);
```

Comment:
I _think_ we do not need to have a separate declaration of the function, since it is never forward-referenced.